src/get-latest.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require('node-fetch');
+
+const fetchJSON = async (url) => {
+	const response = await fetch(url);
+
+	if (response.ok) {
+		return response.json();
+	}
+
+	if (response.status === 404) {
+		return null;
+	}
+
+	return Promise.reject(`The request to "${url}" returned a ${response.status}`);
+};
+
+module.exports = async (packages = []) => {
+	const publishable = packages.filter((pkg) => pkg.private === false);
+	const requests = publishable.map((pkg) => fetchJSON(`https://registry.npmjs.org/${pkg.name}`));
```

Comment:
any reason you're rolling this yourself and not using something like https://www.npmjs.com/package/latest-version? just that you're trying to avoid enormous dependency trees?

Reply:
Yeah, `latest-version` has 57 dependencies whereas `node-fetch` has none!

Reply:
_shakes fist_ SINDREEEEEEEE!!!